libc/test/UnitTest/FPMatcher.h

```diff
@@ -125,6 +210,10 @@ template <typename T> struct FPTest : public Test {
   EXPECT_THAT(actual, LIBC_NAMESPACE::testing::getMatcher<                     \
                           LIBC_NAMESPACE::testing::TestCond::EQ>(expected))
 
+#define EXPECT_CFP_EQ(expected, actual)                                        \
```

Comment:
I have just added the complex variant of EXPECT_EQ for now. I assume we will not be needing all of them and so I think it would be best to implement these as we require them.

Reply:
sgtm